frontend-test/unit/smaaResultsServiceSpec.js

```diff
@@ -454,5 +454,87 @@ define([
         expect(result).toEqual(expectedResult);
       });
     });
+
+    describe('hasNoStochasticMeasurements', function() {
+      it('should return false if there is a performance that has a distribution that is not exact', function() {
+        const aggregateState = {
+          problem: {
+            performanceTable: [{
```

Comment:
I'd have at leat one of these test cases have a performance table with more than 1 entry